osu.Game.Tests/Visual/Navigation/TestSceneSkinEditorNavigation.cs

```diff
@@ -148,7 +148,11 @@ public void TestModOverlayClosesOnOpeningSkinEditor()
 
         private void switchToGameplayScene()
         {
-            AddStep("Click gameplay scene button", () => skinEditor.ChildrenOfType<SkinEditorSceneLibrary.SceneButton>().First(b => b.Text == "Gameplay").TriggerClick());
+            AddStep("Click gameplay scene button", () =>
+            {
+                InputManager.MoveMouseTo(skinEditor.ChildrenOfType<SkinEditorSceneLibrary.SceneButton>().First(b => b.Text == "Gameplay"));
```

Comment:
It's a bit weird repurposing this test in this fashion, bit hard to know why it's done this way without blaming the lines. But I see this as mostly just proving that this PR does fix the issue so won't dwell on it.

Reply:
Yeah, have mentioned in OP that I'm not sure whether there's any point in doing this, but went ahead and did it anyways just to have test coverage.